e2e/interchaintestv8/e2esuite/utils.go

```diff
@@ -94,9 +97,20 @@ func (s *TestSuite) BroadcastMessages(ctx context.Context, chain *cosmos.CosmosC
 // CreateAndFundCosmosUser returns a new cosmos user with the given initial balance and funds it with the native chain denom.
 func (s *TestSuite) CreateAndFundCosmosUser(ctx context.Context, chain *cosmos.CosmosChain) ibc.Wallet {
 	cosmosUserFunds := sdkmath.NewInt(testvalues.InitialBalance)
-	cosmosUsers := interchaintest.GetAndFundTestUsers(s.T(), ctx, s.T().Name(), cosmosUserFunds, chain)
+	cosmosUser := interchaintest.GetAndFundTestUsers(s.T(), ctx, s.T().Name(), cosmosUserFunds, chain)[0]
+
+	// in order to ensure that the underlying account is created, we need to perform an operation on its behalf.
+	// in this case we just send 1 token to itself so the account gets created.
+	err := chain.SendFunds(ctx, cosmosUser.KeyName(), ibc.WalletAmount{
+		Address: cosmosUser.FormattedAddress(),
+		Denom:   chain.Config().Denom,
+		Amount:  sdkmath.NewInt(1),
+	})
+
+	s.Require().NoError(err)
+	s.Require().NoError(testutil.WaitForBlocks(ctx, 2, chain))
```

Comment:
We should also check if we really need the `WaitForBlocks` since this function is ran on a loop in setup. (Shouldn't be too bad tho)